TaskEngine/Tasks/TranscriptionTask.cs

```diff
@@ -52,64 +52,75 @@ protected async override Task OnConsume(string videoId, TaskParameters taskParam
             Video video = await _context.Videos.Include(v => v.Video1).Where(v => v.Id == videoId).FirstAsync();
             Key key = TaskEngineGlobals.KeyProvider.GetKey(video.Id);
 
-            // creat Dictionary and pass it to the recognition function
-            Dictionary<string, List<Caption>> captions = new Dictionary<string, List<Caption>>();
+            video.TranscribingAttempts += 10;
```

Comment:
What's happening here? And in the exception block below `video.TranscribingAttempts += 1000;`?

Reply:
it is a cherry-picked commit that was a minimal change emergency bandaid that was directly applied to production that is now being backported into this PR. I agree it needs a better solution in the future